File: Client/src/app/models/userrow.model.ts
export class UserRowModel {

    constructor( private _Id: string = "", private _Name: string = "", 
                 private _EMail: string = "", 
                 private _SecondName: string = "" ,
                 private _Phone: string = "" ) {
    }

    public get Name(): string {
        
        return this._Name;

    }

    public get EMail(): string {

        return this._EMail;

    }

    public get Id(): string {

        return this._Id;

    }

    public get SecondName(): string {

        return this._SecondName;

    } 

    public get Phone(): string {

        return this._Phone;

    }

}

File: Service/Models/UserRowModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace JobMApi.Models
{
    /// <summary>
    /// 
    /// </summary>
    public class UserRowModel
    {
        /// <summary>
        /// 
        /// </summary>
        [JsonProperty( PropertyName = "name" )]
        public String Name 
        {
            get;
            set;
        }

        /// <summary>
        /// 
        /// </summary>
        [JsonProperty( PropertyName = "email" )]
        public String EMail 
        {
            get;
            set;
        }

        [JsonProperty( PropertyName = "second_name" )]
        public String SecondName
        {
            get;
            set;
        }

        [JsonProperty( PropertyName = "phone" )]
        public String Phone
        {
            get;
            set;
        }

        /// <summary>
        /// 
        /// </summary>
        [JsonProperty( PropertyName = "id" )]
        public String Id 
        {
            get;
            set;
        }
    }
}